src/aws_encryption_sdk/materials_managers/mpl/cmm.py

```diff
@@ -0,0 +1,142 @@
+"""Retrieves encryption/decryption materials from the MPL."""
+
+# These dependencies are only loaded if you install the MPL.
+try:
+    # pylint seems to struggle with this conditional import
+    # pylint: disable=unused-import
+    from aws_cryptographic_materialproviders.mpl.errors import AwsCryptographicMaterialProvidersException
+    from aws_cryptographic_materialproviders.mpl.models import (
+        AlgorithmSuiteIdESDK,
+        CommitmentPolicyESDK,
+        DecryptMaterialsInput,
+        DecryptMaterialsOutput,
+        EncryptedDataKey as MPL_EncryptedDataKey,
+        GetEncryptionMaterialsInput,
+        GetEncryptionMaterialsOutput,
+    )
+    from aws_cryptographic_materialproviders.mpl.references import ICryptographicMaterialsManager
+
+except ImportError:
+    pass
+
+from typing import List
+
+from aws_encryption_sdk.exceptions import AWSEncryptionSDKClientError
+from aws_encryption_sdk.identifiers import CommitmentPolicy
+from aws_encryption_sdk.materials_managers.mpl.materials import MPLEncryptionMaterials, MPLDecryptionMaterials
```

Comment:
Issue/Suggestion: I would be consistent with the prefix. Always use `MPL_ `.

Reply:
These names are a little more nuanced, let me give some more thought to this.
These are classes for the MPL materials that conform to native interfaces.
These internally talk to a `DecryptionMaterials as MPL_DecryptionMaterials` and a `EncryptionMaterials as MPL_EncryptionMaterials` which are actually from the MPL.
If `MPL_` prefix prefixes anything "from the MPL", these shouldn't have the prefix

Reply:
I'm currently at `EncryptionMaterialsFromMPL` and `DecryptionMaterialsFromMPL` but welcome other suggestions

Reply:
I would follow the universal construct of Namespaces before Class names.
i.e: com.amazon.encryption.Encrypt vs Encrypt from com.amazon.Encryption.

I think these are constructs of this library (ESDK-Python)
that wrap constructs from the MPL.

Hence your statement:
> If `MPL_` prefix prefixes anything "from the MPL", these shouldn't have the prefix

However, I disagree with your conclusion.
Just b/c the construct is defined outside of the MPL
(in fact, it is defined in the ESDK-Python's `aws_encryption_sdk.materials_managers.mpl.materials` module)
does not mean it cannot be prefixed with `MPL_`.

I think it is acceptable to prefix anything with `MPL_` if it helps clarify 
that the construct is being introduced/referenced to work with the MPL 
as compared to the existing ESDK-Python.

Reply:
> I think it is acceptable to prefix anything with MPL_ if it helps clarify that the construct is being introduced/referenced to work with the MPL

Agreed but the problem is that there are both
`aws_cryptographic_materialproviders.mpl.models.EncryptionMaterials` that could reasonably called `MPL_EncryptionMaterials`
and
`aws_encryption _sdk.materialsmanagers.mpl.materials.SomeClassNameThatExtends(Native_EncryptionMaterials)`, where `SomeClassNameThatExtends` is currently called `EncryptionMaterialsFromMPL` but could also be called `MPL_EncryptionMaterials`.
I welcome suggestions here since I'm not happy with the current state, but if nobody has alternatives I'd be fine leaving this as-is.

Reply:
You could do something like `Wrapped_MPL`,
to keep everything distinctive... but I agree... at the end of the day, this all pretty confusing...

Maybe we need to do more surgery/refactoring to the ESDK-Python
so that the MPL's materials are a valid extension of the native materials?

That could cut down on the wrapping.